apps/query-ui/queryui/queryui.py

```diff
@@ -90,13 +93,17 @@ def show_dag(plan: LogicalPlan):
     agraph(nodes=nodes, edges=edges, config=config)
 
 
-def run_query(query: str, index: str, plan_only: bool, do_trace: bool):
+def run_query(query: str, index: str, plan_only: bool, do_trace: bool, use_cache: bool):
```

Comment:
UI change suggestion in this block: change `st.text_area("Result", result, height=400)` to
```
st.subheader('Result', divider='rainbow')
st.success(result)
```